pom.xml

```diff
@@ -78,7 +78,7 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <configuration>
-                    <additionalparam>-Xdoclint:none</additionalparam>
```

Comment:
I know Gradle a lot better than Maven. I'm happy to remove these changes if you don't want them.